inc/gtest/gtest_1.8.1.bb

```diff
@@ -0,0 +1,26 @@
+#
+# This recipe builds googletest.
```

Comment:
This .bb file probably should not be in here.

Reply:
Removed from the repo.